src/hotspot/share/gc/parallel/psCardTable.hpp

```diff
@@ -34,6 +34,85 @@ class PSPromotionManager;
 
 class PSCardTable: public CardTable {
```

Comment:
I am aware that `PSCardTable`'s function is not only scavenging support, but I would prefer documentation how this works here (or with the `scavenge_contents_parallel` method, at "Scavenge Support" comment)